packages/studiocms_devapps/src/schema/index.ts

```diff
@@ -11,7 +11,7 @@ export const optionsSchema = z
 		 */
 		endpointPath: z.string().optional().default('/_studiocms-devapps'),
 		/**
-		 * Verbose mode
+		 * Verbose loggin mode
```

Comment:
typo

Reply:
fixed in https://github.com/astrolicious/studiocms/pull/360